electron_app/DownloadManager.js

```diff
@@ -38,4 +61,4 @@ const setHeader = (access_token) => {
   };
 };
 
-module.exports = downloadManager;
+module.exports = { downloadManager, stopAllDownloads, destroyDownloads, stopItemDownload };
```

Comment:
It looks like only `destroyDownloads` is used so there's no need to export `stopAllDownloads()`. Are two methods needed here, or can the work be done in a single method?